libs/core/src/lib/file-uploader/file-uploader.component.ts

```diff
@@ -179,15 +180,24 @@ export class FileUploaderComponent implements ControlValueAccessor {
         }
         const maxSize = this.parseFileSize(this.maxFileSize);
         const minSize = this.parseFileSize(this.minFileSize);
+
         const checkSize = (fileSize: number): boolean => {
-            if (this.maxFileSize && fileSize > maxSize) { return false; }
-            if (this.minFileSize && fileSize < minSize) { return false; }
+            if (this.maxFileSize && fileSize > maxSize) {
```

Comment:
if(this.maxFileSize && fileSize > maxSize || this.minFileSize && fileSize < minSize) {
return false;
}

Reply:
It has been moved to service, I sticked to 2 ifs, because it's easier to read